pilot/pkg/serviceregistry/kube/controller/ambientindex_test.go

```diff
@@ -49,172 +51,208 @@ import (
 	"istio.io/istio/pkg/workloadapi/security"
 )
 
-func TestAmbientIndex(t *testing.T) {
+const (
+	testNS   = "ns1"
+	systemNS = "istio-system"
+	testNW   = "testnetwork"
+	testC    = "cluster0"
+)
+
+func TestAmbientIndex_NetworkAndClusterIDs(t *testing.T) {
```

Comment:
FYI this is a new test. Seemed a reasonable first check to make sure we handle network/cluster names correctly in various events.